examples/qft_with_tn.py

```diff
@@ -67,23 +67,36 @@ def qft(circuit, n):
 # standardize pattern
 pattern = circuit.transpile()
 pattern.standardize()
+nodes, edges = pattern.get_graph()
+
+print(f"Number of nodes: {len(nodes)}")
+print(f"Number of edges: {len(edges)}")
+
+# %%
+# Using efficient graph state simulator `graphix.GraphSim`, we can classically preprocess Pauli measurements.
+
+pattern.shift_signals()
+pattern.perform_pauli_measurements()
+
 nodes, edges = pattern.get_graph()
 g = nx.Graph()
 g.add_nodes_from(nodes)
 g.add_edges_from(edges)
+print(f"Number of nodes: {len(nodes)}")
+print(f"Number of edges: {len(edges)}")
 np.random.seed(100)
-nx.draw(g)
+pos = nx.spring_layout(g)
+nx.draw(g, pos=pos, node_size=15)
```

Comment:
the way you chose to plot the graph makes viewers wonder why they're separated. we should either:
- explain why, or
- change the layout